projects/freedesktop.org/libslirp/package.yml

```diff
@@ -0,0 +1,48 @@
+distributable:
+  url: https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v{{ version }}/libslirp-v{{ version }}.tar.gz
+  strip-components: 1
+
+versions:
+  - 4.7.0
+
+dependencies:
+  gnome.org/glib: ^2
+
+build:
+  dependencies:
+    mesonbuild.com: ^0.63
+    ninja-build.org: 1
+    freedesktop.org/pkg-config: ^0.29
+    tea.xyz/gx/cc: c99
+  script: |
+    meson build $ARGS
+    ninja -C build install all
+  env:
+    ARGS:
+      - -Ddefault_library=both 
+      - --prefix={{ prefix }}
+      - --libdir={{ prefix }}/lib
+      - --buildtype=release
+      - --wrap-mode=nofallback
+
+test:
+  dependencies:
+    tea.xyz/gx/cc: c99
+  script: |
+    mv $FIXTURE test.c
+    gcc test.c -lslirp -o test
+    ./test
+  fixture: |
+    #include <stdio.h>
+    #include <string.h>
+    #include <stddef.h>
+    #include <slirp/libslirp.h>
```

Comment:
This suggests the build script should end with:
```sh
cd "{{prefix}}/lib"
mv slirp/* .
rmdir slirp
ln -s . slirp
```

Reply:
I’ve been doing this when I find situations where other packages just do `#include <libslirp.h>` indicating they expected the headers to be installed un-namespaced.

I’m not sure we should always do it… dunno.